vnext/Microsoft.ReactNative/HStringHelper.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <pch.h>
+
+#include "HStringHelper.h"
+#include "Unicode.h"
+
+#include <winrt/base.h>
+
+namespace winrt {
+
+std::string HSTRINGToString(HSTRING hstring) {
```

Comment:
>HSTRINGToString [](start = 12, length = 15)

Why not to use the winrt built-in functions? See the DynamicWriter/DynamicReader where I use them:
winrt::to_hstring
winrt::to_string